test/integration/singlecluster/controller/dra/dra_controller_test.go

```diff
@@ -0,0 +1,589 @@
+/*
```

Comment:
We don't have a dra_controller anymore, right?

Reply:
updated